src/main/resources/META-INF/resources/primefaces/datatable/datatable.js

```diff
@@ -530,6 +531,7 @@ PrimeFaces.widget.DataTable = PrimeFaces.widget.DeferredWidget.extend({
 
             var columnHeader = $(this),
             sortOrderData = columnHeader.data('sortorder'),
+                // TODO here we should exclude unsorted based on cfg
```

Comment:
Is the TODO meaning you still need to do it?

Reply:
Yes, still working on it in my fork